app/src/main/java/com/example/fitappa/Profile/SetupInformation.java

```diff
@@ -0,0 +1,94 @@
+package com.example.fitappa.Profile;
+
+import java.io.Serializable;
+
+/**
+ * This class is an entity class containing extra information about a user that can be found on their profile.
+ *
+ * The methods in this class are all setters and getters
+ *
+ * The documentation is very basic mainly since all methods are getters and setters
+ *
+ * @author Souren
+ * @since 0.1
+ *
+ */
+
+
+public class SetupInformation implements Serializable {
```

Comment:
remove public access modifiers from all public methods and class declaration (to make it package-protected)